src_docs/version-printer.js

```diff
@@ -1,4 +1,4 @@
-var RSS_VERSION = "1.0.17";
+"var RSS_VERSION = \"1.0.18\";"
```

Comment:
probably another windows issue with the build tools.  any chance you can get this file to not have the escaping?
If not I can merge the PR without it.  pull it locally and re-run